common/OpTestUtil.py

```diff
@@ -1518,7 +1518,7 @@ def get_login(self, host, term_obj, pty, prompt):
                 if rc == 0:
                     pty.sendline(my_pwd)
                     time.sleep(0.5)
-                    rc = pty.expect(['login: $', ".*#$", ".*# $", ".*\$", "~ #",
+                    rc = pty.expect(['login: $', ".*#$", ".*# $", ".*\$", "~ #", ":~",
                                      'Petitboot', pexpect.TIMEOUT, pexpect.EOF], timeout=10)
                     if rc not in [1, 2, 3, 4]:
```

Comment:
when prompt increases than the rc count also increases rite ?

Reply:
pushed the changes , thanks @abdhaleegit for review